task_router_channel.go

```diff
@@ -0,0 +1,83 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+const TaskChannelPathPart = "TaskChannels"
+
+type TaskChannelService struct {
+	client       *Client
+	workspaceSid string
+}
+
+type TaskChannel struct {
+	AccountSid              string            `json:"account_sid"`
+	DateCreated             TwilioTime        `json:"date_created"`
+	DateUpdated             TwilioTime        `json:"date_updated"`
+	FriendlyName            string            `json:"friendly_name"`
+	Sid                     string            `json:"sid"`
+	UniqueName              string            `json: "unique_name"`
+	WorkspaceSid            string            `json:"workspace_sid"`
+	ChannelOptimizedRouting bool              `json:"channel_optimized_routing"`
+	URL                     string            `json:"url"`
+	Links                   map[string]string `json:"links"`
+}
+
+type TaskChannelPage struct {
+	Page
+	TaskChannels []*Worker `json:"task_channels"`
```

Comment:
hmm I'm not sure if this is right, I think this type should be `TaskChannel` instead of Worker.

Reply:
Good catch 💯